src/main/java/seedu/address/model/person/StatusTypes.java

```diff
@@ -19,4 +19,16 @@ public String toString() {
         return this.statusName;
     }
 
+    /**
+     * Checks if the provided status is a valid status.
+     *
+     * @param input The string status to be checked for validity.
+     * @return {@code true} if the status is valid, {@code false} otherwise.
+     */
+    public static boolean isValidStatusType(String input) {
```

Comment:
Have you considered using the given `StringUtil.containsWordIgnoreCase` method to check if the given input is a valid status to ignore any case differences?

i.e.

```
public static boolean isValidStatus(String status) {
        List<String> validStatus = Arrays.asList("Preliminary", "Interviewed", "Rejected", "Offered");
        return validStatus.stream()
                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(status, keyword));
}
```

Reply:
.toLowerCase is already being used so shouuuuld be fine?